### DISCUSSION TITLE
v0.7

### DISCUSSION THREAD

**Discussion by dail8859** [dail8859/NotepadNext#511] [New Releases]
## What's Changed
* Add app to Window's "Open With" suggestions for text files
* Add match/case keywords to Python
* Add ability to set GUI translation via preferences dialog
* Add preference option to set GUI translation
* Add visual indicator of zoom level
* Add persistent bookmarks between sessions
* Add QuickFind wrap indicator
* Style python decorator attributes
* Make language in preference dialog more consistent
* Update Window title to indication if running in admin mode
* Fix file filter on file dialog boxes
* Fix ActionScript
* Keep File List items in order
* Properly handle cancelling of Export As dialog
* Chinese Simplified Translation Update

**Full Changelog**: https://github.com/dail8859/NotepadNext/compare/v0.6.4...v0.7

<hr /><em>This discussion was created from the release <a href='https://github.com/dail8859/NotepadNext/releases/tag/v0.7'>v0.7</a>.</em>

**Comment by Ser-J-6936** [comment]
Dear developer, tell me, is there an opportunity to change to a dark theme?

**Reply by dail8859** [reply]
Currently it is not implemented. There is a PR which started to add support for dark mode: https://github.com/dail8859/NotepadNext/pull/411

**Comment by amsharma9** [comment]
@dail8859 Request you to kindly implement Dark mode asap. With your expertise it might, at most, be a weekend of effort for you to check the PR, so kindly try. It would help a lot. Secondly, I wanted to know if there is any keyboard short cut to traverse between tabs. Notepad++ has Ctrl+Tab. Does it work in NotepadNext?

**Reply by dail8859** [reply]
Its way more than just "checking a PR". And its nice of you to assume I have a free weekend's worth of time :) There are new changes coming built into Qt that will likely help with dark mode. There are also 3rd party libraries that may be an option. So alot of unknowns that I haven't had time to investigate at all. The absolute one thing I don't want to do is add dark mode in a way, and then it becomes a burden to maintain properly.

> I wanted to know if there is any keyboard short cut to traverse between tabs. Notepad++ has Ctrl+Tab

It is implemented and will be available in the next release.

**Reply by amsharma9** [reply]
Not to take away any credit for building a great utility, thanks for putting in all the effort. I understand that the PR is probably not straightforward. I am a programmer, if I can help you in any way, kindly let me know. Although, I haven't worked in C++ for a long time, and don't know Qt.

**Comment by grompes** [comment]
do not work ubuntu 24.04.
setup ubuntu. open Appstore, search and install Notepad Next, ... open.. no open,.. delete =)

**Reply by dail8859** [reply]
Sorry to hear that.
